packages/editor/src/toolbar/popups/link-popup.tsx

```diff
@@ -64,19 +64,41 @@ export function LinkPopup(props: LinkPopupProps) {
         }}
       >
         {!isImageActive && (
-          <Input
-            type="text"
-            placeholder={strings.linkText()}
-            defaultValue={link.current?.title}
-            sx={{ mb: 1 }}
-            onChange={(e) =>
-              (link.current = {
-                ...link.current,
-                title: e.target.value
-              })
-            }
-          />
+          <>
+            <Text
+              sx={{
+                mb: 1,
+                ml: 1,
+                fontSize: "body",
+                color: "accent"
```

Comment:
I don't think we should be using the `accent` color here.

Reply:
What color would you suggest?